en/components/general/ignite-ui-licensing.md

```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
 Update the azure-pipelines.yml with the following steps:
 
 ```cmd
-npm config set @infragistics:registry $(npmRegistry)
-```
+pool:
+  vmImage: 'Ubuntu 16.04'
 
-```cmd
-npm config set always-auth true --scope:@infragistics
-```
+steps:
+
+- script: echo "@infragistics:registry=$(npmRegistry)" >> ~/.npmrc
+  displayName: 'Npm add registry'
+
+- script: echo "$(igScope):always-auth=true" >> ~/.npmrc
+  displayName: 'Npm config'
+
+- script: echo "$(igScope):_auth=$(token)" >> ~/.npmrc
+  displayName: 'Npm config auth'
+
+- script: npm install
+  displayName: 'Install dependencies'
+  env:
+    AZURE_PIPELINES: "true"
+
+- script: npm run lint
+  displayName: 'Run lint'
+
+- script: npm run build
+  displayName: 'Generate live editing and build samples'
 
-```cmd
-npm config set _auth=$(token) --scope:@infragistics
 ```
 
-<img class="responsive-img" style="-webkit-box-shadow: 8px 9px 9px 5px #ccc; -moz-box-shadow: 8px 9px 9px 5px #ccc; box-shadow: 8px 9px 9px 5px #ccc; min-width: calc(100% - 450px)" 
-  src="../../images/general/azure-ci-pipelines-ci-yml-3.jpg"
-  data-src="../../images/general/azure-ci-pipelines-ci-yml-3.jpg"
-  alt="Azure Pipelines CI yml update"
-  title="Azure Pipelines CI yml update" />
+Now we need to add variables for the *npm registry*, *scope* and *token*. There are two ways to do so:
 
-Add **npm registry** and **token** variables.
+ #### [Define Variable Group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml) from the Library page under Pipelines.
```

Comment:
Oh boy, using small headers as emphasis and for entire sentences with links. You can bold.. or put a single line of text under.